content/chapter-05/SetupEx.md

```diff
@@ -20,14 +21,18 @@ sudo apt-get update && sudo apt-get install logstash
 * -e: enable specifying configuration directly from the command line
 * stdin: file handle that process reads to get information from you, human
 * stdout: process writes log information to this file handle
-* After pipeline has started, type:
+* After pipeline has started  
+```
+The stdin plugin is now waiting for input:
```

Comment:
The stdin plugin is now waiting for an input

Reply:
This is actually a message from logstash:
```
root@ubuntu2:/usr/share/logstash# sudo bin/logstash -e 'input { stdin { } } output { stdout {} }'
WARNING: Could not find logstash.yml which is typically located in $LS_HOME/config or /etc/logstash. You can specify the path using --path.settings. Continuing using the defaults
Could not find log4j2 configuration at path /usr/share/logstash/config/log4j2.properties. Using default config which logs errors to the console
The stdin plugin is now waiting for input:
Hello World!
2018-01-06T17:03:01.282Z ubuntu2 Hello World!
```